### DISCUSSION TITLE
Sound Speed Manager configuration with SIS 5

### DISCUSSION THREAD

**Discussion by cpeters220** [oceanmapping/community#43] [Community Announcements and Awareness]
There is **one more step** that needs to be taken to properly configure SIS5 to work with Sound Speed Manager. This step has been updated in the online SSM documentation: 

https://www.hydroffice.org/manuals/soundspeed/stable/user_manual_setup_sis_v5.html#sis-v5-settings

In SIS 5, Under _Tools_ in _Parameter Setup/Logging_, locate the _Disable DDIST if 1_ entry.

If the value of ‘**Disable DDIST if 1**’ is 1, **change the value to 0** and **restart SIS**.

![Enable DDIST in Parameter Setup Logging control](https://github.com/oceanmapping/community/assets/88506899/86f061a4-0f2b-4ac5-9c98-76f7a90b78e1)

**Comment by rolker** [comment]
How do we resolve a "(DDS) Not able to create DDist process. Try to restart. (652)" error when restarting SIS after making this change? On two similar systems both running SIS 5.11.1, one works fine while the other generates that error.

**Reply by kjerram** [reply]
Any updates?  Would you like me to post this as an Issue and see if others have feedback?

**Reply by cpeters220** [reply]
I suggest updating to SIS 5.14 when it is published in a few weeks because it resolves this issue. A new, dedicated datagram is configured in SIS to send data to SSM which streamlines this process. 

Are you using TCP for data from PU (two settings down) on both machines?
